src/main/java/com/google/sps/servlets/RecipeServlet.java

```diff
@@ -37,4 +58,39 @@ public void doPut(HttpServletRequest request, HttpServletResponse response) thro
   @Override
   public void doDelete(HttpServletRequest request, HttpServletResponse response)
       throws IOException {}
+
+  private void getRecipeList(HttpServletResponse response) throws IOException {
+    Query query = DBReferences.recipes;
+    ApiFuture<QuerySnapshot> querySnapshot = query.get();
+    ArrayList<Object> recipeList = new ArrayList<>();
+
+    try {
+      for (DocumentSnapshot document : querySnapshot.get().getDocuments())
+        recipeList.add(document.getData());
```

Comment:
We prefer having {} even for one liners. This way you find out you need another line in the loop, you won't need to remember adding them.